data/nickaldwin.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Nick Aldwin D. Lemoncito",
+  "type": "personal",
+  "bio": "Full-stack Developer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/nicklemoncito",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/nickaldwin",
+      "icon": "FaGithub"
+    },
```

Please remove this trailing comma `,` from here. 

```suggestion
    }
```

---

ok sir thank you so much